### DISCUSSION TITLE
context deadline exceeded

### DISCUSSION THREAD

**Discussion by ari123mster** [prometheus/snmp_exporter#1078] [Q&A]
hello i want monitoring my mikrotik but i have problem in snmp_exporter

Get "http://192.168.100.111:9116/snmp?auth=public_v2&module=mikrotik&target=192.168.100.254": context deadline exceeded

can you help me 

thanks

**Comment by JorianBorst** [comment]
This means that getting the data from your snmp agent takes too long. This can be fixed by giving it more time, in Prometheus this is done by setting the scrape_timeout. Another option would be to trim the snmp.yml file.
